README.md

```diff
@@ -1,20 +1,27 @@
-# Packer Builder for Anka
+# Packer Plugin for Anka
 
 This is a [Packer Builder] for building images that work with [Veertu Anka], a macOS virtualization tool.
 
 Note that this builder does not manage images. Once it creates an image, it is up to you to use it or delete it.
 
+### v2.0.0 Breaking Changes
+
+* Plugin will only work with Packer v1.7 or later.
+* Plugin has been renamed from `packer-builder-veertu-anka` to `packer-plugin-veertu-anka`.
+* Builder has been renamed from `veertu-anka` to `veertu-anka-vm`.
+
 ### Compatibility
 
-Packer Version | Builder for Anka Version
+Packer Version | Veertu Anka Plugin Version
 --- | ---
 Up to 1.4.5 | 1.1.0
 1.5.x and above | 1.2.0
+1.7.x and above | 2.0.0
```

Comment:
We can probably remove the older versions from the table since the plugin name will be changing. We'll archive `packer-builder-veertu-anka`